Slim/Utils/Misc.pm

```diff
@@ -860,6 +860,12 @@ sub folderFilter {
 	my @path = splitdir(shift);
 	my $folder = pop @path;
 
+	# Skip this folder (and subfolders) if there is a file .slimskip.
+	if ( -f catdir(@path, $folder, ".slimskip") ) {
+		$scannerlog->error("Skipping: " . catdir(@path, $folder));
```

Comment:
Please make this a conditional `if main::INFOLOG && $scannerlog->is_info && $scannerlog->info(...)`.

I'm wondering about the performance penalty. It's one more read on every folder.

Reply:
Done.

I also added a preference setting.

I'm not afraid for performance penalties. It is just a stat of a name in a directory that is being processed and therefore already in memory.